pkg/runctl/controller.go

```diff
@@ -244,37 +244,40 @@ func (c *Controller) syncHandler(key string) error {
 	}
 
 	runManager := c.createRunManager(pipelineRun, pipelineRunsConfig)
-	// Process pipeline run based on current state
-	switch state := pipelineRun.GetStatus().State; state {
-	// TODO fix #117
```

Comment:
fixes #117
(comment should close the issue)

Reply:
added to description

Reply:
`#177` refers to an issue in the Git repository we used before moving here.